### ISSUE TITLE
Chain theme errors with Elvish 0.13 with default prompts

### ISSUE THREAD

**Issue by vendion** [zzamboni/elvish-themes#5]
Using commit https://github.com/zzamboni/elvish-themes/commit/5cd2d7c0ec9d705195ae730f89ca3844d4f12c38 with Elvish v0.13 and the default prompts (both left and right). Example below where I originally launch the shell without the theme then call `use` to try to load it:

```
/usr/home/vendion> use github.com/zzamboni/elvish-themes/chain
/usr/home/vendion> read: Illegal option -d
prompt function error: sh exited with 2
(prompt here but not rendered)
read: Illegal option -d                                                                                                                                                                                                                        
prompt function error: sh exited with 2
(prompt here but not rendered)
```
I have even tried to manually define the segmants for the left and right prompts with the following and still get the same error:

```
chain:prompt-segments = [ su dir git-branch git-combined arrow ]
chain:rprompt-segments = [ (constantly (styled (whoami)✸(hostname) inverse)) timestamp ]
```
but I still get the same error as above.

**Comment by zzamboni** [comment]
@vendion thanks for the report!

This error is produced by the href/elvish-gitstatus library (used by the git segments) and is caused by an incompatibility in the `sh` command, see https://github.com/href/elvish-gitstatus/issues/3.

The best fix for this is to update to Elvish master and apply the patch from https://github.com/href/elvish-gitstatus/pull/5 to use the new `read-upto` function instead of the external shell call.

Alternatively, you can leave things as they are and just replace `sh` with `bash` in https://github.com/href/elvish-gitstatus/blob/master/gitstatus.elv#L217

I'll close this issue since this is not an issue in the chain theme.

**Comment by vendion** [comment]
Thanks for getting back to me so quickly, I just did the dirty fix for now as I was testing out Elvish 0.13 before updating the FreeBSD port for it.

I do really like your theme none the less, thank you for your great work ;)

**Comment by zzamboni** [comment]
Happy you could fix it, and thanks for the kind words!
